web-ng/src/app/components/form-field-editor/form-field-editor.component.scss

```diff
@@ -19,6 +19,7 @@
 .field-type {
   margin-left: 30px;
   font-size: 12px;
+  margin-top: -3px;
```

Comment:
Negative margins are valid CSS, but they may make the layout harder to reason about. Is it possible to reduce the margin of one the question label divs instead?